proposals/acp-specification/acp.ttl

```diff
@@ -0,0 +1,244 @@
+    rdfs:comment "The ACP deny predicate defines the set of access modes denied by a policy if satisfied, it has a range of acp:AccessMode and a domain of acp:Policy."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:domain acp:Policy ;
+    rdfs:range acp:AccessMode .
+
+acp:attribute
+    a rdf:Property ;
+    rdfs:label "attribute"@en ;
+    rdfs:comment "An attribute describing resource access. The ACP acp:attribute predicate has a range of rdfs:Resource and a domain of acp:Context."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:domain acp:Context ;
+    rdfs:range rdfs:Resource .
+
+acp:resource
+    a rdf:Property ;
+    rdfs:label "resource"@en ;
+    rdfs:comment "A resource subjected to ACP access control. ACP resource is a sub property of acp:attribute."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:subPropertyOf acp:attribute .
+
+acp:agent
+    a rdf:Property ;
+    rdfs:label "agent"@en ;
+    rdfs:comment "An agent trying to access a resource. ACP agent is a sub property of acp:attribute."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:subPropertyOf acp:attribute .
+
+acp:creator
+    a rdf:Property ;
+    rdfs:label "creator"@en ;
+    rdfs:comment "An agent credited as the creator of a resource. ACP creator is a sub property of acp:attribute."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:subPropertyOf acp:attribute .
+
+acp:owner
+    a rdf:Property ;
+    rdfs:label "owner"@en ;
+    rdfs:comment "An agent credited as the owner of a resource. ACP owner is a sub property of acp:attribute. "@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:subPropertyOf acp:attribute .
+
+acp:client
+    a rdf:Property ;
+    rdfs:label "client"@en ;
+    rdfs:comment "A client application used to access a resource. ACP client is a sub property of acp:attribute."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:subPropertyOf acp:attribute .
+
+acp:issuer
+    a rdf:Property ;
+    rdfs:label "issuer"@en ;
+    rdfs:comment "An identity provider credited as the issuer of the credentials that asserted the identity of the agent trying to access a resource. ACP issuer is a sub property of acp:attribute."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:subPropertyOf acp:attribute .
+
+acp:time
+    a rdf:Property ;
+    rdfs:label "time"@en ;
+    rdfs:comment "A time at which a resource is accessed. ACP time is a sub property of acp:attribute."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:subPropertyOf acp:attribute .
+
+acp:vc
+    a rdf:Property ;
+    rdfs:label "vc"@en ;
+    rdfs:comment "A verifiable credentials that is presented as part of a resource access request. ACP vc is a sub property of acp:attribute."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:subPropertyOf acp:attribute .
+
+acp:mode
+    rdfs:label "mode"@en ;
+    rdfs:comment "The ACP mode predicate defines the set of granted access modes for resource access, it has a range of acp:AccessMode and a domain of acp:Context."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:domain acp:Context ;
```

Comment:
5.1. Options discovery uses this predicate in HTTP Link header

I think in that case this `rdfs:domain` would entail that any `<>` being the [`Response.url`](https://developer.mozilla.org/en-US/docs/Web/API/Response/url) that uses `http://www.w3.org/ns/solid/acp#mode` in the response HTTP Link header

```ttl
<> a acp:Context
```

Again I'm not sure if that is intentional.

Reply:
Context being defined as descriptions of resource access seems to match with that to me, but I'm happy to discuss the semantics of Context.

Something like "those modes are in context for accessing that resource".

Reply:
Here we get this resource itself **is a** Context